settings.py

```diff
@@ -33,7 +33,10 @@ def read_bool(val: str) -> bool:
 REDIS_PASS = os.environ["REDIS_PASS"]
 REDIS_USE_SSL = read_bool(os.environ["REDIS_USE_SSL"])
 
+OSU_API_KEYS = os.environ["OSU_API_KEYS"].split(",")
```

Comment:
@tsunyoku is your beatmaps service ready for this kind of "fetch by id" request? I'd love to avoid spreading out these API keys more than we already have -- would be good to limit them to this new beatmaps service